spec/request/upload_supporting_evidence_spec.rb

```diff
@@ -9,10 +9,31 @@
     context 'with valid parameters' do
       it 'returns a 200 and an upload guid' do
         post '/v0/upload_supporting_evidence',
-             params: JSON.parse('{"supporting_evidence_attachment": {"file_data": "filename"}}')
+             params: { supporting_evidence_attachment: { file_data: fixture_file_upload('files/sm_file1.jpg') } }
         expect(response).to have_http_status(:ok)
         expect(JSON.parse(response.body)['data']['attributes']['guid']).to eq SupportingEvidenceAttachment.last.guid
       end
+
+      it 'returns a 422  for a malformed pdf' do
```

Comment:
@Mottie 
example error details are: 
`The uploaded PDF file is invalid and cannot be read`
`The file uploaded was too large`
`You are not allowed to upload "crt" files, allowed types: pdf, png, gif, tiff, tif, jpeg, jpg, bmp, txt`

The last one will be a bigger pain to change because it comes from a gem.

Reply:
Is there a spec for file size too large?

Reply:
kind of, yes.  There's a spec that covers the size_range  range error https://github.com/department-of-veterans-affairs/vets-api/pull/5059/files#diff-e1189c09092bfde373ce490438c2dabbR38

Reply:
but beyond that I'd say carrierwave specs cover the behavior of their validator.